terraform/deployments/github/repos.yml

```diff
@@ -298,6 +298,16 @@ repos:
         - Format
         - Type checks
 
+  govuk-chat-gradio-prototype:
+    homepage_url: "https://docs.publishing.service.gov.uk/apps/govuk-chat-gradio-prototype.html"
+    required_status_checks:
+      standard_contexts: *standard_security_checks
```

Comment:
We can't have these because of the private repo.

Reply:
oh ofc sorry i did this before the seal PR. Will remove them.